bitbots_quintic_walk/include/bitbots_quintic_walk/walk_stabilizer.h

```diff
@@ -16,8 +17,10 @@ class WalkStabilizer : public bitbots_splines::AbstractStabilizer<WalkResponse>
  public:
   WalkStabilizer();
   virtual void reset();
-  virtual std::unique_ptr<bio_ik::BioIKKinematicsQueryOptions> stabilize(const WalkResponse &positions);
+  std::unique_ptr<bio_ik::BioIKKinematicsQueryOptions> stabilize(const WalkResponse &response, const ros::Duration &dt) override;
```

Comment:
please use trailing return type for this function means 'std::unique_ptr<bio_ik::BioIKKinematicsQueryOptions>' should be replaceable by 'auto' which makes its way better readable.

Reply:
we don't want to use auto